interface-definitions/high-availability.xml.in

```diff
@@ -252,6 +252,144 @@
+              <valueHelp>
+                <format>u32:1-600</format>
+                <description>Interval in seconds (default: 10)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-3600"/>
+              </constraint>
+            </properties>
+            <defaultValue>10</defaultValue>
+          </leafNode>
+          <leafNode name="forward-method">
+            <properties>
+              <help>Forwarding method (default - NAT)</help>
+              <completionHelp>
+                <list>direct nat tunnel</list>
+              </completionHelp>
+              <valueHelp>
+                <format>direct</format>
+                <description>Direct routing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>nat</format>
+                <description>NAT</description>
+              </valueHelp>
+              <valueHelp>
+                <format>tunnel</format>
+                <description>Tunneling</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(direct|nat|tunnel)$</regex>
+              </constraint>
+            </properties>
+            <defaultValue>nat</defaultValue>
+          </leafNode>
+          #include <include/port-number.xml.i>
+          <leafNode name="persistence-timeout">
+            <properties>
+              <help>Timeout for persistent connections</help>
+              <valueHelp>
+                <format>u32:1-86400</format>
+                <description>Timeout for persistent connections (default 300)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-86400"/>
+              </constraint>
+            </properties>
+            <defaultValue>300</defaultValue>
+          </leafNode>
+          <leafNode name="protocol">
+            <properties>
+              <help>Protocol for port checks (default TCP)</help>
+              <completionHelp>
+                <list>tcp udp</list>
+              </completionHelp>
+              <valueHelp>
+                <format>tcp</format>
+                <description>Protocol TCP</description>
+              </valueHelp>
+              <valueHelp>
+                <format>udp</format>
+                <description>Protocol UDP</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(tcp|udp)$</regex>
+              </constraint>
+            </properties>
+            <defaultValue>tcp</defaultValue>
+          </leafNode>
+          <tagNode name="real-server">
+            <properties>
+              <help>Real server address</help>
+            </properties>
+            <children>
+              #include <include/port-number.xml.i>
+              <leafNode name="connection-timeout">
+                <properties>
+                  <help>Connection timeout to remote server</help>
```

Comment:
I'd reword to "Server connection timeout".

Reply:
Done